docs/registration.adoc

```diff
@@ -60,17 +60,17 @@ Command templates for the supported applications:
 
 [source,bash]
 ----
-export KEEP_CLIENT_CONFIG_DIR=$(pwd)/config
-
 export KEEP_CLIENT_ETHEREUM_WS_URL="<Ethereum API WS URL>"
+export OPERATOR_ADDRESS="<Operator Account Address>"
+
+export KEEP_CLIENT_CONFIG_DIR="<Path to the directory containing the Staking Provider Key File>"
 
-export STAKING_PROVIDER_KEY_FILE_PASSWORD="<Staking Provider Account Key File Password>"
 export STAKING_PROVIDER_KEY_FILE_NAME="<Staking Provider Account Key File Name>"
-export OPERATOR_ADDRESS="<Operator Account Address>"
+export STAKING_PROVIDER_KEY_FILE_PASSWORD="<Staking Provider Account Key File Password>"
```

Comment:
Two late comments here:
1. I believe `export` is not needed. We use environment variables only to make this example easy to follow and explain the values.
    ```
    $ echo $ETHEREUM_WS_URL

    $ ETHEREUM_WS_URL="ws://my.host/ws"
    $ echo $ETHEREUM_WS_URL
    ws://my.host/ws
2. The only environment variable accepted by Keep Client is `KEEP_ETHEREUM_PASSWORD`. Everything else is in the config. I think we should remove all `KEEP_CLIENT_*` prefixes from these environment variables to not suggest these environment variables are read by the client. I was highly confused about them for a moment so I expect all readers will be even more confused.

To sum up, I would make it look as follows:
```
ETHEREUM_WS_URL="<Ethereum API WS URL>"
OPERATOR_ADDRESS="<Operator Account Address>"
CONFIG_DIR="<Path to the directory containing the Staking Provider Key>"
STAKING_PROVIDER_KEY_FILE_NAME="<Staking Provider Account Key File Name>"
STAKING_PROVIDER_KEY_FILE_PASSWORD="<Staking Provider Account Key File Password>"

docker run ...
```

Reply:
e5da9de9d42f6a1b647d68f3a529fc561e607c0a

I left `KEEP_` prefix for config and storage variables, as we want to be clear and not confuse them in case other applications are being set up on a machine.

Reply:
I would also remove `KEEP_` prefix for config and storage variables. We do not export them so they are set only for the current terminal session. And it's just a simplification for docs, no one even has to set them, they can pass values to flags directly. 

If we leave `KEEP_` for some of them, this will IMO cause confusion as well. Operator Account is also only for Keep, and Staking Provider Account is also only for Keep. Regarding the conflicts, I imagine `ETHERUM_WS_URL` could be conflicting as well. 

We also still have at least one occurrence of `KEEP_CLIENT_ETHEREUM_WS_URL` in docs.

Reply:
As you wish Lord Commander 11d1cb3b9b087e32c0582f05de88a1c4eae70ecf